bio/bowtie2/align/wrapper.py

```
@@ -4,10 +4,104 @@
+
+    # Sort alignments using picard SortSam.
+    pipe_cmd = (
+        " | picard SortSam {java_opts} {sort_extra} "
+        "--INPUT /dev/stdin "
+        "--TMP_DIR {tmpdir} "
+        "--SORT_ORDER {sort_order} "
+        "--OUTPUT {snakemake.output[0]}"
+    )
+
+else:
+    raise ValueError(f"Unexpected value for params.sort ({sort})")
```

Ruff (0.8.0)

Undefined name `sort`

(F821)